partials/home-press-quotes.php

```diff
@@ -0,0 +1,62 @@
+<?php
+// Get Home page
+$home = get_page_by_path('home');
```

Comment:
I don't really like this pattern for data here. The home page's content is never needed or displayed the home page is just a place where metavalues are set? Seems cleaner to me to just have that meta set in the site options. This is obviously a point of discussion